application/common/models/Mfa.php

```diff
@@ -441,6 +454,7 @@ public static function getTypes()
             self::TYPE_MANAGER => 'Manager Backup Code',
             self::TYPE_TOTP => 'Smartphone App',
             self::TYPE_U2F => 'Security Key',
+            self::TYPE_WEBAUTHN => 'Security Key',
```

Comment:
Might be helpful to use a different label so it will be easier to see which ones are U2F and which ones are WebAuthn. Maybe just "Key" or "Hardware Key"? I don't know how long it will take for all instances to be converted over to WebAuthn

Reply:
For instance, this shows in Sherlock
![image](https://user-images.githubusercontent.com/3172830/144654993-ca898f11-5e28-4e12-9f88-813890a7800b.png)

Reply:
I was just trying to keep it transparent in any way that users might see, though I'm not sure if these values are displayed to end users. maybe in the profile review module?